fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -3073,6 +3083,9 @@ public ParseNode visitModifyTablePropertiesClause(StarRocksParser.ModifyTablePro
     @Override
     public ParseNode visitAddColumnClause(StarRocksParser.AddColumnClauseContext context) {
         ColumnDef columnDef = getColumnDef(context.columnDesc());
+        if (columnDef.isAutoIncrement()) {
```

Comment:
You can output the column name in the exception.

Reply:
have fixed it